docs/install.md

```diff
@@ -6,7 +6,7 @@ Start by installing these programs:
  * [Steam](http://steampowered.com)
  * [nodejs](http://nodejs.org)
  * [Github client](http://desktop.github.com/)
- * [Sublime Text 3](http://www.sublimetext.com/) **optional**
```

Comment:
Don't remove ST3. A lot of people are using it.
And if you're at it add Atom Editor as well.